core/admin/starter-sites/src/library/container.js

```
@@ -0,0 +1,641 @@
+import TypesenseInstantSearchAdapter from 'typesense-instantsearch-adapter/src/TypesenseInstantsearchAdapter.js';
+import {
+	InstantSearch,
+	SearchBox,
+	connectMenu,
+	connectHierarchicalMenu,
+	connectCurrentRefinements,
+	Stats,
+	Configure,
+} from 'react-instantsearch-dom';
+import classnames from 'classnames';
+import { isEmpty, unescape } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "unescape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)